backend/medtagger/repositories/users.py

```diff
@@ -43,3 +44,11 @@ def set_user_info(user: User, firstName: str, lastName: str) -> None:
             user.first_name = firstName
             user.last_name = lastName
             session.add(user)
+
+    @staticmethod
+    def set_user_settings(name: str, value: object) -> None:
+        """Set user's settings parameter of specified name to provided value."""
+        with db_session() as session:
+            settings = get_current_user().settings
+            setattr(settings, name, value)
```

Comment:
I've received an exception here :( User's Settings were empty - set to None.

![screen shot 2018-05-11 at 21 09 00](https://user-images.githubusercontent.com/6514553/39943156-2613223a-5562-11e8-8dfc-2b70a29a788b.png)